### PR TITLE
CSS updates for Publisher

### PR DESCRIPTION
**Pull request by roderickmorales**
- Updated CSS to match UI design
- Changed the tags around for <main> and <nav> because they were reversed
- Added a condition in base template to apply a different layout for dashboard vs the rest of the app

### CODE REVIEW

File: course_discovery/static/sass/publisher/grid.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.col-60{
+	width: 60%;
+}
+
+.col-40{
+	width: 40%;
+}
```
Comment (waheedahmed) [inline course_discovery/static/sass/publisher/grid.scss]: Add new line at end of file.

File: course_discovery/static/sass/publisher/layout.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+.layout-col-b{
+	background: $gray-background;
+}
+
+//at the momment main-ltr is adding a margin of 20px at the bottom of the div
+.container .wrapper-footer{
+	margin: 0 !important;
+	background: $white;
+}
```
Comment (waheedahmed) [inline course_discovery/static/sass/publisher/layout.scss]: Same here, add new line.

File: course_discovery/static/sass/publisher/lib.scss [hunk 3]
```diff
@@ -9,6 +9,9 @@
   }
 }
 
+.float-left{ float: left; }
```
Comment (waheedahmed) [inline course_discovery/static/sass/publisher/lib.scss]: RTL support for this and next line.

File: course_discovery/static/sass/publisher/publisher.scss [hunk 4]
```diff
@@ -30,21 +35,29 @@
...
+            width: 37%;
         }
 
         .layout-col-b-custom {
             @include margin-right(0);
             width: 55%;
+            background: $white;
+            padding-top: 1em;
+            padding-bottom: 1em;
+            padding-right: 2.5%;
```
Comment (waheedahmed) [inline course_discovery/static/sass/publisher/publisher.scss]: Add RTL support for both this and next line.

**Review by waheedahmed — APPROVED** [review]
Minor comments, 👍 once addressed.

**Comment by awais786** [comment]
@roderickmorales whats the plan for the forms ?

**Comment by awais786** [comment]
@roderickmorales I have rebased.

**Pull request merged**